uikit/buttonV2/Button.tsx

```diff
@@ -36,13 +39,13 @@ const buttonClasses = {
     variant: {
         primary: "bg-dark-neutral-primary text-dark-background-default hover:bg-dark-neutral-soft",
         secondary: "bg-light-neutral-subtle/10 text-dark-neutral-primary hover:bg-light-neutral-subtle/20",
-        alternate: "bg-dark-primary text-white hover:bg-dark-primary-soft",
+        alternate: "bg-dark-primary text-white hover:bg-light-darker",
         discord: "bg-discord-dark text-white hover:bg-discord-light",
         twitter: "bg-twitter-dark text-white hover:bg-twitter-light",
```

Comment:
The bg color for variant `discord` and `twitter` is reversed between the default and hover state sir (default should be light, hover should be dark)
![image](https://user-images.githubusercontent.com/81855912/175076210-db01606b-7616-4e4c-a5e6-e31ec5f3ff64.png)

Reply:
:( :( :( :( :( :( :( :( :(